src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/SystemIPGlobalStatistics.cs

```diff
@@ -12,17 +12,19 @@ internal sealed class SystemIPGlobalStatistics : IPGlobalStatistics
 
         private SystemIPGlobalStatistics() { }
 
-        internal SystemIPGlobalStatistics(AddressFamily family)
+        internal unsafe SystemIPGlobalStatistics(AddressFamily family)
         {
-            uint result = Interop.IpHlpApi.GetIpStatisticsEx(out _stats, family);
+            uint result;
+            fixed (Interop.IpHlpApi.MibIpStats* pStats = &_stats)
```

Comment:
I think libraries usually does the braces for fixed (even single line).

Reply:
Done.